### PR TITLE
Added Node.js help page

### PR DESCRIPTION
**Pull request by akalipetis**
You can view this at: https://4fv6pjnu.apps.lair.io/stacks/nodejs/

### CODE REVIEW

File: _stacks/nodejs.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+3. Your server script - defined in `package.json` - is called `start`
+  * if you use another script for development, i.e. `dev`, you need to change it in the prjoect settings, available using the gear icon next to your project name
+
+### Libraries
+
+If you need any libraries, you can easily install them using NPM - either
+using the fully featured Linux Terminal of SourceLair, or the Command Palette
+command.
+
+### `package.json` `scripts`
```
Comment (parisk) [inline _stacks/nodejs.md]: I propose removing the code quotes. Styling seems messed up.

File: _stacks/nodejs.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+2. Your application's server file is called app.js
+  * if you've named it anything else, you need to add the correct project root in the prjoect settings, available using the gear icon next to your project name
+3. Your server script - defined in `package.json` - is called `start`
+  * if you use another script for development, i.e. `dev`, you need to change it in the prjoect settings, available using the gear icon next to your project name
+
+### Libraries
+
+If you need any libraries, you can easily install them using NPM - either
+using the fully featured Linux Terminal of SourceLair, or the Command Palette
+command.
```
Comment (parisk) [inline _stacks/nodejs.md]: Should we refer that all they have to do is type "npm install" in the command palette?

File: _stacks/nodejs.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Cloning
+
+If you're cloning an existing application, you just need to make sure the
+following prerequisites exist, in order to get it working with your Public URL
+properly.
+
+1. Your application is listening to **port 3000**, which is the default one for Express.js and most frameworks
+2. Your application's server file is called app.js
+  * if you've named it anything else, you need to add the correct project root in the prjoect settings, available using the gear icon next to your project name
```
Comment (parisk) [inline _stacks/nodejs.md]: > in the prjoect settings

project settings

Also, I am no sure how the proper project root has to do with the `app.js` file naming. E.g. what if my file is called `server.js` and is placed in the project root dir.

Comment (akalipetis) [inline _stacks/nodejs.md]: We need this in order to determine the project root. I just thought of a nicer way to do so - search for the package.json file - so I'll remove this and push an update to SourceLair.

File: _stacks/nodejs.md (hunk 2 above)
Comment (akalipetis) [inline _stacks/nodejs.md]: Yes, changed it.

File: _stacks/nodejs.md (hunk 1 above)
Comment (akalipetis) [inline _stacks/nodejs.md]: You are correct, removed them.

**Pull request merged**